samples/excel/06-named-item/create-and-remove-named-item.yaml

```diff
@@ -0,0 +1,164 @@
+id: excel-named-item-create-and-remove-named-item
+name: Create & remove a named item
+description: Create & remove a named item for a formula
+author: Microsoft
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#add-name").click(addName);
+        $("#remove-name").click(removeName);
+
+        async function addName() {
+            try {
+                await Excel.run(async (context) => {
+
+                    await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Sample");
```

Comment:
I would like us to adopt the pattern of having a setup function that provides the setup of the Sample sheet and add sample data (in this case the table) and then a shorter function that demonstrates the API. Look at get / set values for a range snippet for an example. That way it becomes more clear what code is needed to use the API and what code is setup code.